.github/workflows/ci.yml

```diff
@@ -22,54 +22,38 @@ jobs:
       - name: Noop
         run: "true"
   checks:
-    name:  TOXENV=${{ matrix.tox-env }}
+    name:  TOXENV=format-check,typecheck,vendor-check,package -- --additional-format sdist --additional-format wheel
```

Comment:
Just helping clear space. All Combined this is ~5 minutes whereas the fastest test shards are 8. With the 3 job headspace cleared, the overall job count went from 23 -> 27. Previously 4 mac shards, now 5.